web/server/utils.py

```diff
@@ -104,3 +104,22 @@ def is_relative_to(path: PurePath, other: PurePath | str) -> bool:
         return True
     except ValueError:
         return False
+
+
+def ensure_list(value: t.Optional[t.Any] = None) -> t.List:
```

Comment:
Can we use ensure_list from from sqlglot.helper? I see it used in various places in our codebase.